vector/src/main/java/im/vector/riotx/core/extensions/ViewExtensions.kt

```diff
@@ -48,3 +48,10 @@ fun EditText.showPassword(visible: Boolean, updateCursor: Boolean = true) {
     }
     if (updateCursor) setSelection(text?.length ?: 0)
 }
+
+fun View.getMeasurements(): Pair<Int, Int> {
```

Comment:
Create a data class to avoid using generic pair? (give sense to first and second)

Reply:
This is just a utility method, I don't think it's worth it